AN00209_xCORE-200_DSP_Library/app_complex_fir/src/app_complex_filters.xc

```diff
@@ -8,36 +8,38 @@
 #include <dsp.h>
 
 dsp_complex_t data[4] = {
-    {1000,-1000},
-    {500,500},
-    {-8000,1000},
-    {100,0},
+    {1000, -1000},
+    {500, 500},
+    {-8000, 1000},
+    {100, 0},
 };
 
 dsp_complex_t fir[4] = {
-    {0x0100000,-0x0100000},
+    {0x0100000, -0x0100000},
     {0x1000000, 0x1000000},
-    {0x2000000,-0x1000000},
-    {-0x0400000,0x0200000},
+    {0x2000000, -0x1000000},
+    {-0x0400000, 0x0200000},
 };
 
 int main(void) {
-    timer tmr;
-    int t0, t1;
-tmr :> t0;
-    dsp_complex_t x = dsp_complex_fir(data, fir, 4, 0, 24);
-tmr :> t1;
-//    printf("%d\n", t1-t0);
-    dsp_complex_t sum = {0,0};
-    for(int i = 0; i < 4; i++) {
-        sum = dsp_complex_add(sum, dsp_complex_mul(data[i], fir[i], 24));
-    }
-    printf("%d, %d    %d,%d\n", x.re, x.im, sum.re, sum.im);
-    if (x.re == sum.re && x.im == sum.im) {
-        printf("PASS\n");
-    } else {
-        printf("FAIL\n");
-    }
-            
-    return 0;
+  timer tmr;
+  int   t0, t1;
+tmr:
```

Comment:
Unfortunately you seem to have used a version of clang-format that doesn't support XC.
For this reason it has interpreted `tmr:>t0;` as being a label no a read from a timer.